webservices/args.py

```diff
@@ -615,6 +615,23 @@ def _deserialize(self, value, attr, data):
     'is_notice': fields.List(fields.Bool, description='Record filed as 24- or 48-hour notice'),
 }
 
+schedule_e_efile = {
+    #'cycle': fields.List(fields.Int, description=docs.RECORD_CYCLE),
```

Comment:
No need for cycle on this?  I noticed the `report_year` commented out in the factory in the tests above too, so just wanted to double check.